index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <meta content="width=device-width, initial-scale=1.0" name="viewport">
+  <title>Set up and Mobile Version Skeletion</title>
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" />
+
+  <link rel="stylesheet"
+    href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+  <link rel="stylesheet" href="styles.css" />
+</head>
+
+<body>
+  <header>
+    <nav class="nav-bar">
+      <a href="#">My Logo</a>
+      <i class="fas fa-bars"></i>
+    </nav>
+  </header>
+  <main>
+    <section>
+      <div>
+        <div id="background-image"></div>
+        <div class="content">
+          <h1>I’m Esi Ogagaoghene <br />
+            Glad to see <br /> you!
+          </h1>
+          <p>
+            I’m a software developer! I can help you build a product , feature or website Look through some of my work
+            and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+          </p>
```

Comment:
## Optional Change

- [ ] Kindly add spacing between the heading and the `p` tag I marked in red on the screenshots below. Let there be a little distance just as in the Figma template. Please make that change before working on your next tasks. 👍 

**See your design**
![image](https://user-images.githubusercontent.com/50941074/170491248-6e53df82-46ab-445f-8264-0e6a2bb3e961.png)

**The Figma template design**
![image](https://user-images.githubusercontent.com/50941074/170491396-be4605e8-6c50-44c9-8dff-4936833ecfe5.png)

Reply:
Thanks for your professional review